Issue #09

lhqhuy14032001 opened the pull request:
I was performed following your guide to create the terms and conditions page. I hope that help your issue.

JaedonSpurlock01 commented:
You're on the right track. Could you replace the underlines with our organization "GDSC-CSUSM".

lhqhuy14032001 commented:
I replaced the underlines with your organization.

On Fri, Feb 7, 2025 at 5:00 PM Jaedon Spurlock ***@***.***>
wrote:

> You're on the right track. Could you replace the underlines with our
> organization "GDSC-CSUSM".
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/csusmGDSC/csusmgdsc-web/pull/12#issuecomment-2642471345>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVVAXK2KC26APFYD5YDMJ2D2OR72NAVCNFSM6AAAAABWVLABD6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNBSGQ3TCMZUGU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

lhqhuy14032001 commented:
I replaced the underlines. Please, review and let me know your thoughs
On Sat, 8 Feb 2025 at 08:32 Lê Hoàng Quốc Huy ***@***.***>
wrote:

> I replaced the underlines with your organization.
>
> On Fri, Feb 7, 2025 at 5:00 PM Jaedon Spurlock ***@***.***>
> wrote:
>
>> You're on the right track. Could you replace the underlines with our
>> organization "GDSC-CSUSM".
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/csusmGDSC/csusmgdsc-web/pull/12#issuecomment-2642471345>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AVVAXK2KC26APFYD5YDMJ2D2OR72NAVCNFSM6AAAAABWVLABD6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNBSGQ3TCMZUGU>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

JaedonSpurlock01 reviewed (commented):
lgtm

JaedonSpurlock01 commented:
Good job on your first PR. Welcome to the open-source community
